README.md

```diff
@@ -257,6 +260,26 @@ and `.dsc` files, do this:
 
 Note that we no longer actively try to get it into the Debian repository.
 
+Building the Windows Installer
+---------------------------
+
+You will need a Debian-ish operating system, and the following packages:
+
+```
+apt-get install build-essentials gcc-mingw-w64-i686 libgcrypt-mingw-w64-dev \
```

Comment:
Should this be suggesting 32bit deps instead of 64bit? (or can you not specify the w64 bit and let apt pick the most appropriate one)